packages/office-ui-fabric-react/src/components/Callout/Callout.scss

```diff
@@ -42,7 +42,7 @@ $Callout-smallbeak-slant-width: 16px;
 .main {
   background-color: $ms-color-white;
   overflow-x: hidden;
-  overflow-y: auto;
```

Comment:
Would you mind providing a before and after screenshot? I just wan't to make sure this fixes the problem. Thank you!!

Reply:
![2017-11-11 1](https://user-images.githubusercontent.com/28269448/32693662-2fca2bfe-c6f4-11e7-8147-73a86510c98c.png)
![2017-11-11](https://user-images.githubusercontent.com/28269448/32693663-34435fde-c6f4-11e7-8b01-94971787f0fc.png)

Reply:
When I change the this is what I get on my local machine when I do the changes